### ISSUE TITLE
Postgres failed to parse query: GENERATED ALWAYS AS

### ISSUE THREAD

**Issue by sk-** [sbdchd/squawk#172]
The following statement is not parseable
```sql
ALTER TABLE table_c ADD column c boolean GENERATED ALWAYS AS (p IS NOT NULL) STORED NOT NULL;
```

```
Problem linting SQL files: Postgres failed to parse query
```
# Version
```
$ squawk --version
squawk 0.8.1
```

**Comment by chdsbd** [comment]
Squawk uses the Postgres query parser from Postgres 10 via [libpg_query-sys](https://github.com/tdbgamer/libpg_query-sys). I think we need to update libpg_query-sys to use Postgres 12 or later to fix this issue.

**Comment by sbdchd** [comment]
Yeah latest libpg_query version supports Postgres

https://github.com/pganalyze/libpg_query/commit/d1d0186b80c1fbf496265789af89da4e7ca890ab

**Comment by chdsbd** [comment]
This issue has been fixed in 0.8.2, which can be installed from NPM.

**Comment by sk-** [comment]
Thanks @chdsbd for the quick fix. It's great to see such a responsive team in an Open Source project.
